CONTRIBUTING.md

```diff
@@ -4,25 +4,32 @@
 
 Feel free to raise an issue if you have a question, an enhancement, or a bug report.
 
-Use the issue search functionality to search all **_opened and closed_** issues before raising a new issue. If you raise a duplicate issue, you're just creating noise for everyone watching this repo.
+Use the issue search functionality to search all **_opened and closed_** [issues](https://github.com/typescript-eslint/typescript-eslint/issues) before raising a new issue. If you raise a duplicate issue, you're just creating noise for everyone watching this repo.
 
 Before raising a bug, ensure you are using the latest version of our packages. We release every week, so there's a good chance your issue might have already been fixed.
 
-Finally, when raising a new issue, please fill out the issue template - **_please don't skip sections_**.
+Finally, when raising a new issue, please fill out the [issue template](https://github.com/typescript-eslint/typescript-eslint/issues/new/choose) - **_please don't skip sections_**.
 
 Please provide **_as much information as possible_**. This project is maintained by volunteers, so the more information you provide, the less likely we will have to waste everyone's time in asking you for more information.
 
-If you have a particularly complex issue - consider creating a small, self-contained reproduction repo. This will help you in figuring out the exact problem, and will help us in reproducing and diagnosing the bug.
+If you have a particularly complex issue - consider creating a small, self-contained reproduction repo or use our [playground](https://typescript-eslint.io/play). This will help you in figuring out the exact problem, and will help us in reproducing and diagnosing the bug.
```

Comment:
The complex issue template is meant for things that can't be demonstrated on the playground. +1 to mentioning it though. How does this strike you?

```suggestion
If you have a particularly complex issue that can't be reproduced on [our playground](https://typescript-eslint.io/play) - consider creating a small, self-contained reproduction repo or use. This will help you in figuring out the exact problem, and will help us in reproducing and diagnosing the bug.
```